Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.3.0
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential nodejs && apt-get clean
+
+ENV DATABASE_URL mysql2://collections_pub@mysql/collections_publisher_development
```

Comment:
I see this is tied into: https://github.com/alphagov/publishing-e2e-tests/blob/add-collections/docker-compose.yml#L20-L22 which seems to be a bit more specific than other apps e.g: https://github.com/alphagov/publishing-api/blob/master/Dockerfile#L4 

And I wonder if that will make it more difficult for a second mysql app?

Is it possible to just run the mysql as a default mysql user and not need those constants in the docker-compose.yml?

Reply:
From what I've seen while adding this is it tries to create a database for the URL provided, I could try passing the user instead of as an ENV variable and see if the root user can work around these problems. Then we'd just need to set the root password instead.

Reply:
Yeah part of `db:setup` is the creating of the database(s).

So with postgres image we have a default user (postgres) with an empty password and we can proceed with that pretty simply. If mysql is the same we might be able to just have `mysql2://mysql@mysql/collections_publisher` and not need the mysql env vars?